index.d.ts

```diff
@@ -48,6 +48,34 @@ declare namespace prettyBytes {
 		```
 		*/
 		readonly binary?: boolean;
+
+		/**
+		Number of significant digits to display when calculating bytes or bits.
+
+		@default 3
+
+		```
+		import prettyBytes = require('pretty-bytes');
+
+		prettyBytes(1019, {digits: 3});
```

Comment:
It makes no sense for the example to use the default value. That's just confusing.